src/main_dpdk.cpp

```diff
@@ -910,6 +922,23 @@ COLD_FUNC static int parse_options(int argc, char *argv[], bool first_time ) {
             case OPT_SLEEPY_SCHEDULER:
                 CGlobalInfo::m_options.m_is_sleepy_scheduler = true;
                 break;
+            case OPT_LATENCY_MEASUREMENT_METHOD:
```

Comment:
Could we  move this configuration to RPC command? CLI is overloaded and should be used only if it is something that we can't change dynamically

Reply:
I do assume latency measurement or time synchronization will not be changed dynamically.